smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/AddEventStreamDependency.java

```diff
@@ -99,6 +103,31 @@ public Map<String, Consumer<TypeScriptWriter>> getRuntimeConfigWriters(
         }
     }
 
+    @Override
+    public List<? extends CodeInterceptor<? extends CodeSection, TypeScriptWriter>> interceptors(
+        TypeScriptCodegenContext codegenContext
+    ) {
+        return List.of(CodeInterceptor.appender(SmithyContextCodeSection.class, (w, s) -> {
+            EventStreamIndex eventStreamIndex = EventStreamIndex.of(s.getModel());
+            boolean input = eventStreamIndex.getInputInfo(s.getOperation()).isPresent();
+            boolean output = eventStreamIndex.getOutputInfo(s.getOperation()).isPresent();
+            // If not event streaming for I/O, don't write anything
+            if (!input && !output) {
+                return;
+            }
+            // Otherwise, write present input and output streaming
+            w.writeDocs("@internal");
```

Comment:
the internal annotation has no meaning here since this is not an accessible field